dotnet/metadata.yaml

```diff
@@ -2,18 +2,16 @@
 #   (such as test files) with just the path and no services block.
 # Otherwise, they'll be picked up as not yet cleaned.
 #
-# ---
-# High-value example 1
-# files:
-#  - path: <file path relative to this metadata.yaml>
-#    services:
-#      - s3
-#      - sqs
-#  - path: <file 2>
-#    services:
-#      - dynamodb
+---
+#Code example Getting Started
```

Comment:
Getting started with Visual Studio code example?

Reply:
That's the internal metadata file - not really customer-facing, even though they can see it.